.github/workflows/kubernetes-tentacle-publish-chart.yaml

```diff
@@ -0,0 +1,45 @@
+name: Publish Chart
+
+on:
+  pull_request:
+  
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+          
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: 'v3.13.2'
+
+      - name: Parse Chart config
+        uses: pietrobolcato/action-read-yaml@1.0.0
+        id: read_chart_yaml
+        with:
+          config: ${{ github.workspace }}/charts/kubernetes-tentacle/Chart.yaml
+
+      - name: Get branch names
+        id: branch_names
+        uses: OctopusDeploy/util-actions/current-branch-name@current-branch-name.0.1.0
+
+      - id: version
+        run: |
+          echo "CHART_VERSION=${{ steps.read_chart_yaml.outputs.version }}-${{ steps.branch_names.outputs.branch_name }}-$(date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
+          
+      #- name: Login to Artifactory
+      #  run: helm registry login packages.octopushq.com -u '${{ secrets.ARTIFACTORY_USERNAME }}' -p '${{ secrets.ARTIFACTORY_PASSWORD }}'
```

Comment:
Please remove the endpoint here and use a secret

Reply:
done.